server/xpub/entities/manuscript/resolvers.js

```diff
@@ -91,21 +80,22 @@ const resolvers = {
         throw new Error(errorManuscript)
       }
 
-      const manuscript = Manuscript.applyInput(
+      const manuscript = ManuscriptManager.applyInput(
         originalManuscript,
         manuscriptInput,
       )
 
       manuscript.status = 'QA'
-      await Manuscript.save(manuscript)
+      await ManuscriptManager.save(manuscript)
 
-      mecaExport(manuscript.id, user, ip)
+      const content = 'dummy content'
+      mecaExport(manuscript, content, ip)
         .then(() =>
           logger.info(`Manuscript ${manuscript.id} successfully exported`),
         )
         .catch(async err => {
           logger.error('MECA export failed', err)
-          await Manuscript.save({
+          await ManuscriptManager.save({
             id: manuscript.id,
             status: 'FAILED_MECA_EXPORT',
```

Comment:
If this is where `FAILED_MECA_EXPORT` is set on the manuscript - where is succeeded equivalent set ?

Reply:
Currently nowhere. I think there's a separate piece of work to make more sense of the statuses. Currently it gets set to `QA` on finishSubmission and will stay that way if this doesn't fail.